### PR TITLE
Travis CI setup

### PR DESCRIPTION
**Pull request by manuelq12**
Configuration for Travis CI

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,152 @@
-### Ignore for NodeJs
-node_modules
```
Comment (mateoescobarm) [inline .gitignore:2]: deleting this from gitignore will make travis also thats code we dont really need too be up here

Comment (manuelq12) [reply .gitignore:2]: I added up in line 42

**Review by mateoescobarm — APPROVED** [review]
LGTM

**Review by mateoescobarm — APPROVED** [review]
LGTM

**Review by mateoescobarm — APPROVED** [review]
LGTM

**Pull request merged**